back/app/services/input_status_service.rb

```diff
@@ -1,12 +1,47 @@
 # frozen_string_literal: true
 
 class InputStatusService
+  AUTOMATED_TRANSITIONS = {
```

Comment:
I wonder of it still makes sense to keep this constant.

Reply:
Agree it's pretty useless. It wouldn't be if it also defined the full behavior and conditionality, but since those are implemented in `auto_transition_input!` anyway, we might as well make the implementation of that method fully explicit.

But it's not that important, think it's really okay to keep it as it, it's simple enough.